### ISSUE TITLE
For building spaceEngineers do we need to buy the game

### ISSUE THREAD

**Issue by Eulerianial** [KeenSoftwareHouse/SpaceEngineers#550]
I am trying to build SpaceEngineers and during the build, it says I need to buy SpaceEnigeers game first. Am  I right?

**Comment by Jimmacle** [comment]
Yes, the game isn't open source so you still need to buy it through Steam.

**Comment by Eulerianial** [comment]
Actually I was building the space engineers from source to work locally. In the Marek's blog I read that people can build the game and customize it. So isn't it open source yet.

**Comment by Jimmacle** [comment]
It isn't open source because people aren't free to use and redistribute the software without arranging it with Keen first (if they're still offering grants for total conversions). The source is available so SE players can contribute improvements to the game, not to take it for their own use.

**Comment by aaronfranke** [comment]
"It isn't open source because people aren't free to use and redistribute the software" That's called "Free software" not "open source".

The real reason you need to own SE is that it's not completely open-source, most of the game is but the executable is not buildable.

**Comment by Jimmacle** [comment]
No, the game is not open source whatsoever.

**Comment by miad2401** [comment]
The code is here but not the assets, which is why you need to buy it.

**Comment by debian-user-france1** [comment]
Close please
